### PR TITLE
Update index.md

### PR DESCRIPTION
**Pull request by leonardoporro**
May I add my tool? Thanks!

### CODE REVIEW

File: entity-framework/core/extensions/index.md [hunk 1]
```diff
@@ -235,3 +235,11 @@ An implementation for soft deleting entities. For EF Core: 3.
 Extends EF Core to resolve connection strings from App.config. For EF Core: 3.
 
 [GitHub repository](https://github.com/efcore/EFCore.ConfigurationManager)
+
+### Detached
+
+A DTO-Entity mapper with composition/aggregation handling (similar to GraphDiff), for EF Core 3.1+.
```
Comment (ErikEJ) [inline entity-framework/core/extensions/index.md:241]: Please use the same version indication as all other entries.

File: entity-framework/core/extensions/index.md [hunk 2]
```diff
@@ -235,3 +235,13 @@ An implementation for soft deleting entities. For EF Core: 3.
 Extends EF Core to resolve connection strings from App.config. For EF Core: 3.
 
 [GitHub repository](https://github.com/efcore/EFCore.ConfigurationManager)
+
+### Detached
+
+A DTO-Entity mapper with composition/aggregation handling (similar to GraphDiff). For EF Core: 3
```
Comment (ErikEJ) [inline entity-framework/core/extensions/index.md:241]: Almost: 3.

File: entity-framework/core/extensions/index.md [hunk 3]
```diff
@@ -235,3 +235,9 @@ An implementation for soft deleting entities. For EF Core: 3.
 Extends EF Core to resolve connection strings from App.config. For EF Core: 3.
 
 [GitHub repository](https://github.com/efcore/EFCore.ConfigurationManager)
+
+### Detached
```
Comment (smitpatel) [inline entity-framework/core/extensions/index.md:239]:
```suggestion
### Detached Mapper
```

File: entity-framework/core/extensions/index.md [hunk 4]
```diff
@@ -235,3 +235,9 @@ An implementation for soft deleting entities. For EF Core: 3.
...
+
+A DTO-Entity mapper with composition/aggregation handling (similar to GraphDiff). For EF Core: 3, 5.
+
+[NuGet](https://www.nuget.org/packages/Detached.Mappers.EntityFramework)
```
Comment (smitpatel) [inline entity-framework/core/extensions/index.md:243]:
```suggestion
[NuGet](https://www.nuget.org/packages/Detached.Mappers.EntityFramework)

[GitHub repository](https://github.com/leonardoporro/Detached-Mapper)

```

**Pull request merged**